pagopa/io-app
# chore: [IOCOM-1358] FIMS flow IAB saga

## Short description
Addition of the required saga for LolliPoP auth and IAB url gathering for FIMS

## List of changes proposed in this pull request
- small mock update 
- addition of handleFimsGetRedirectUrlAndOpenIAB saga
- addition of its required utility functions
- tests to come

## How to test
1) check out [this dev-server branch](https://github.com/pagopa/io-dev-api-server/tree/feature/fims)
2) in the dev-server's config.ts file, set `withCTA` to `true`
3) start a FIMS flow from the messages_home route
4) make sure it all goes swimmingly and at the end of the flow the app navigates back and opens the IAB
